docker-compose.yml

```diff
@@ -19,6 +19,14 @@ services:
       - /home/node/app/node_modules
       - ./api:/home/node/app
 
+  scholly:
```

Comment:
Not related to the work on this PR, but can we map port 3000 in client container to 3000 on the host, I think it would be easier to just have the same link to the website regardless of whether you're running locally or in docker. 

Plus, for some reason the website doesn't load (times out) if I use the other address assigned by docker and not the usual `http://localhost:3000`

Reply:
You should use the nginx proxy locally which exposes `3050`